exporter/pwd_file.go

```diff
@@ -0,0 +1,31 @@
+package exporter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type PasswordMap struct {
+	RedisPwd map[string]string
+}
+
+func NewPasswordMap() *PasswordMap {
```

Comment:
No need for this extra function, instead the caller can simply do `&PasswordMap{}`